### ISSUE TITLE
Does "Segment Anything" support using scripts to segment a certain type of objects in a large number of images?

### ISSUE THREAD

**Issue by 0xfocu5** [facebookresearch/segment-anything#176]
Does "Segment Anything" support using scripts to segment a certain type of objects in a large number of images? Real-time detection, inference, and bounding box drawing, similar to the YOLO series.

If so, could you provide some tutorial cases? Thank you, looking forward to your reply.

**Comment by susigo** [comment]
same question here

**Comment by Yuqifan1117** [comment]
You can focus on this annotation pipeline. https://github.com/Yuqifan1117/Annotation-anything-pipeline
https://github.com/facebookresearch/segment-anything/issues/142#issue-1660266003

**Comment by 0xfocu5** [comment]
> You can focus on this annotation pipeline. https://github.com/Yuqifan1117/Annotation-anything-pipeline [#142 (comment)](https://github.com/facebookresearch/segment-anything/issues/142#issue-1660266003)

I don't think this is quite suitable for me. I just want to infer a specific category of object in an image, rather than all of them, such as people passing by in real-time video surveillance.

**Comment by Yuqifan1117** [comment]
> 

It also can infer a specific category of object in an image just given the specific label word in '--specific_label'. For more fine-grianed semantic labels, we will use other object detectors to refine in the future. For more fine-grained labels, you can also see in https://github.com/fudan-zvg/Semantic-Segment-Anything

**Comment by AzulYang** [comment]
Same question. by the way, the current SAM saves each instance. Can we only segment the certain of objects and save one mask, with only the type of objects on this mask?

**Comment by Karol-G** [comment]
You can create click-based semantic and instnce segmentations with SAM with this repo [napari-sam](https://github.com/MIC-DKFZ/napari-sam)

**Comment by flameflows** [comment]
I think this is helpful
https://github.com/fudan-zvg/Semantic-Segment-Anything
